chart/compass/templates/tests/director/director-test.yaml

```diff
@@ -93,20 +93,62 @@ spec:
               value: "https://{{ .Values.global.gateway.tls.secure.oauth.host }}.{{ .Values.global.ingress.domainName }}/director/graphql"
             - name: APP_WEBHOOK_URL
               value: "https://kyma-project.io"
+            - name: APP_DEFAULT_SCENARIO_ENABLED
```

Comment:
Why did we want to enable/disable the Default scenario?

Reply:
As discussed, this is needed because e2e tests assume that this is enabled by default. But on real envs this is disabled